akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -271,15 +275,18 @@ final case class ContactPoints(contactPoints: Set[ActorPath]) {
     contactPoints.asJava
 }
 
+@deprecated("Use Akka gRPC instead.", since = "2.6.0")
 object ClusterClient {
 
   /**
    * Scala API: Factory method for `ClusterClient` [[akka.actor.Props]].
    */
+  @deprecated("Use Akka gRPC instead.", since = "2.6.0")
```

Comment:
Put the url of migrate-to-akkka-grpc in the docs in the deprecation message for easy finding it?